tests/acceptance/features/bootstrap/TestingAppContext.php

```diff
@@ -558,6 +558,56 @@ public function theAdministratorDeletesTheDirectoryUsingTheTestingApi($dir) {
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * get log file using the testing API
+	 *
+	 * @return void
+	 */
+	public function getLogfile() {
+		$response = OcsApiHelper::sendRequest(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getAdminUsername(),
+			$this->featureContext->getAdminPassword(),
+			'GET',
+			$this->getBaseUrl("/logfile"),
+			[],
+			$this->featureContext->getOcsApiVersion()
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * clear the contents of the logfile using the testing API
+	 *
+	 * @return void
+	 */
+	public function clearLogFile() {
+		$response = OcsApiHelper::sendRequest(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getAdminUsername(),
+			$this->featureContext->getAdminPassword(),
+			'DELETE',
+			$this->getBaseUrl("/logfile"),
+			[],
+			$this->featureContext->getOcsApiVersion()
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @Then log entries should decrease when the administrator clears the logfile
+	 *
+	 * @return void
+	 */
+	public function logEntriesShouldDecreaseWhenTheAdministratorClearsTheLogfile() {
+		$this->getLogfile();
+		$initialCount = \count($this->featureContext->getResponseXml()->data[0]);
```

Comment:
I guess we assume that the installation and startup will have put something in the log file, so ``initialCount`` will be enough greater than 0.